src/OpenTK.NT/Native/User32/Structs/DeviceMode.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System.Runtime.InteropServices;
+
+using DWORD = System.UInt32;
+using WORD = System.UInt16;
+
+namespace OpenTK.NT.Native
+{
+    [StructLayout(LayoutKind.Sequential, CharSet = CharSet.Auto)]
+    public struct DeviceMode // unions aren't represented
```

Comment:
Unions can be represented via StructLayout and FieldOffset - thoughts on this struct?